projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.ts

```diff
@@ -80,10 +83,20 @@ export class BulkActionsDropdownComponent implements OnInit, OnDestroy {
         this.tooltip = title;
         this.placeholder = placeholder;
       });
+
+    this.extensions.resetBulkActionsSubject$.pipe(takeUntil(this.onDestroy$)).subscribe(() => {
```

Comment:
could you please avoid adding `Subject` to the end of property, `$` is enough

Reply:
Fixed, thanks!